src/app/components/vertical-tabs/index.html

```diff
@@ -20,16 +20,44 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Vertical tab properties">
+    <sky-demo-page-property
+      propertyName="tabHeading"
+    >
+    Specifies the tab header.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="active"
       isOptional="true"
     >
-    Indicates whether the tab is active when the tabset loads.
+    Indicates whether the tab is active when the tabset loads. This property accepts <stache-code>boolean</stache-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="tabHeading"
+      propertyName="ariaControls"
+      isOptional="true"
     >
-    Specifies the tab header.
+    Specifies an ID to identify the element that contains the content the vertical tab exposes. This property supports <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure">accessibility rules for disclosures</a>. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="ariaInvalid"
+      isOptional="true"
+    >
+    Specifies which values are invalid in the tab's content when the tabset is part of a form. This property accepts <stache-code>boolean</stache-code> values.
```

Comment:
To follow the style pattern with boolean properties, change "Specifies which" to "Indicates whether." I'd also add "the" before "values." ... And based on the note in, I think I'd move "are invalid" after "content." I think it reads more naturally that way, and I think this wording changes the meaning slightly (although I might be mistaken about that).